index.html

```diff
@@ -0,0 +1,11 @@
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<meta http-equiv="X-UA-Compatible" content="ie=edge">
+	<title>Odd numbers</title>
+</head>
+<body>
+	<script src="oddNumbers.js"></script>
+</body>
+</html>
```

Comment:
Always add empty line at the end of files.

Reply:
Understand.